src/classic/components/InfoCols/InfoCols.tsx

```diff
@@ -0,0 +1,101 @@
+import { ArrowRightIcon } from "@heroicons/react/24/outline"
+import { HomepageSectionWrapper } from "../HomepageSectionWrapper"
+import InfoColsProps from "~/common/InfoCols"
+
+const InfoColsHeader = ({
+  title,
+  subtitle,
+}: Pick<InfoColsProps, "title" | "subtitle">) => (
+  <div className="flex flex-col gap-4">
+    {subtitle && (
+      <p className="text-subtitle uppercase tracking-widest">{subtitle}</p>
+    )}
+    <h1 className="text-secondary text-5xl font-semibold leading-tight">
+      {title}
+    </h1>
+  </div>
+)
+
+const InfoBoxes = ({ infoBoxes }: Pick<InfoColsProps, "infoBoxes">) => {
+  const mdColsClass =
+    infoBoxes.length === 1 ? "md:grid-cols-1" : "md:grid-cols-2"
+
+  // Follows current behaviour: if there are 4 info boxes, xl screen should stay at 2 columns instead of going up to 3
+  const xlColsClass =
+    infoBoxes.length === 1
+      ? "xl:grid-cols-1"
+      : infoBoxes.length % 2 === 0
+        ? "xl:grid-cols-2"
+        : "xl:grid-cols-3"
+
+  return (
+    <div>
+      <div
+        className={`grid grid-cols-1 gap-4 justify-between ${mdColsClass} ${xlColsClass}`}
+      >
+        {infoBoxes.map((infoBox, idx) => (
+          <div
+            key={idx}
```

Comment:
it's usually advised not to use the idx as the key. can we use any other property of infobox as the key?

Reply:
I think there are 2 ways we can fix this
1. uuid on FE
2. add `id` field to backend

Reply:
can we just use the title as the key?

Reply:
i don't think that's a good idea, from my understanding:
1. react keys should be static/independent of the fields itself; it kinda defeats the point of having a key if when we change the title the entire component has to re-render because the key has changed
2. we also have no guarantees that titles are unique, for e.g current CMS behaviour uses some hardcoded string as title when adding new infoboxes